spec/features/claim_xml_generation_spec.rb

```diff
@@ -264,7 +264,7 @@ def xpath_collection(path)
       end
       it 'has Checksum(s)' do
         expect(xpath_collection("//Files/File/Checksum")).
-          to match_array %w<ee7d09ca06cab35f40f4a6b6d76704a7 58d5af93e8ee5b89e93eb13b750f8301 2e8c57f80b37129030867a4b003c6629>
+          to match_array %w<ee7d09ca06cab35f40f4a6b6d76704a7 58d5af93e8ee5b89e93eb13b750f8301 236ea37419ebc7e70a1a27ea7e43508e>
```

Comment:
Not sure why the checksum has changed. I did a visual inspection of the PDF and both addresses were present and correct.